maven/bnd-export-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -0,0 +1,68 @@
+package aQute.bnd.maven.export.plugin;
+
+import static org.apache.maven.plugins.annotations.LifecyclePhase.PACKAGE;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+import aQute.bnd.build.Container;
+import aQute.bnd.build.StandaloneRun;
+import aQute.bnd.osgi.Jar;
+import biz.aQute.resolve.ProjectResolver;
+
+@Mojo(name = "export", defaultPhase = PACKAGE, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
+public class ExportMojo extends AbstractMojo {
+
+	@Parameter(readonly = true, required = true)
+	private File bndrun;
+
+	@Parameter(defaultValue = "${project.build.directory}", readonly = true)
+	private File targetDir;
+
+	@Parameter(readonly = true, required = false)
+	private boolean resolve = true;
```

Comment:
Yes. I am quite sure that his feelings are wrong :-) ... or at least that there are many cases when the build time resolve works well.
I have created an example project that defines a pretty complete application stack with hibernate, Aries JPA, Aries, Transaction, Aries RSA, CXF, CXF-DOSGi. See https://github.com/cschneider/bndtools-tutorials
The build time resolve works very well and is not that slow. The whole project builds in about 11 seconds including index creation and resolve. The index file has 18.000 lines. So I think this should work quite well in practice.
Still I added a boolean so you can decide if you want to export only or resolve and export.
My main intent here is that people can do a full build on the console without touching bndtools. Of course you will use bndtools to create the requirements but it should also work on plain console.